.github/workflows/run_tests.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   run_tests:
     runs-on: ubuntu-latest
-    if: ${{ ! contains(github.event.head_commit.message, '[skip slow tests]') }}
+    if: ${{ ! contains(github.event.head_commit.message, '[skip fast tests]') }}
```

Comment:
@digicosmos86 I am not sure this is needed. Can this line be dropped?

Reply:
So the setup is now to run all tests? Can `codecov` collect test results from separate runs? In principle I'd want fast tests to run so the tests fail fast

Reply:
@digicosmos86 
> Can codecov collect test results from separate runs?

It seems like the tooling for that is not reliable -- see https://stackoverflow.com/questions/73688076/pytest-with-cov-append-flag-overwrites-coverage-file. Besides, we are using `xdist` to run the tests in parallel. We could mark the slow tests but it's not clear to me that would matter to `xdist`. We can have a fail fast strategy with the `--exitfirst` flag, however. I'll add that.